### PR TITLE
chore(es/minifier): expose `globals_defs` pass

### PR DESCRIPTION
**Pull request by iheyunfei**
**Description:**

I'm trying to do some global expression replacement and found I could reuse this part. Would it be ok to expose this function?

**BREAKING CHANGE:**

**Related issue (if exists):**

### CODE REVIEW

File: crates/swc_ecma_minifier/src/lib.rs [hunk 1]
```diff
@@ -111,14 +111,14 @@ pub fn optimize(
...
 
-        if !defs.is_empty() {
-            let defs = defs.iter().map(|(k, v)| (k.clone(), v.clone())).collect();
-            n.visit_mut_with(&mut global_defs::globals_defs(
-                defs,
-                extra.unresolved_mark,
-                extra.top_level_mark,
-            ));
-        }
+        // if !defs.is_empty() {
```
Comment (kdy1) [inline crates/swc_ecma_minifier/src/lib.rs:114]: Why did you commented this out?

**Review by kdy1 — CHANGES_REQUESTED** [review]

File: crates/swc_ecma_minifier/src/lib.rs (hunk 1 above)
Comment (iheyunfei) [reply crates/swc_ecma_minifier/src/lib.rs:114]: My mistake. I'm debugging the minifier.

**Comment by kdy1** [comment]
Also, do you know https://rustdoc.swc.rs/swc_ecma_transforms_optimization/fn.inline_globals2.html ? 
It's more feature-rich

**Comment by iheyunfei** [comment]
> Also, do you know https://rustdoc.swc.rs/swc_ecma_transforms_optimization/fn.inline_globals2.html ? It's more feature-rich

Yeah. It's a little more complicated than we need.

**Review by kdy1 — APPROVED** [review]
swc-bump:
 - swc_ecma_minifier

**Review by kdy1 — APPROVED** [review]

**Pull request merged**